bin/run-worker.sh

```diff
@@ -1,7 +1,3 @@
 #!/bin/bash -ex
 
-exec newrelic-admin run-program celery -A basket.news worker \
-                                       -P "${CELERY_POOL:-prefork}" \
-                                       -l "${CELERY_LOG_LEVEL:-warning}" \
-                                       -c "${CELERY_NUM_WORKERS:-4}" \
-                                       -Q celery,snitch
+exec python manage.py rqworker --with-scheduler
```

Comment:
we dropping new relic?

Reply:
Nope, missed that. My eyes were searching for `celery`. I'll make a note to add that back in.